modules/apps/ai-creator/ai-creator-openai-api/src/main/java/com/liferay/ai/creator/openai/util/AICreatorOpenAIURLHelper.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.ai.creator.openai.util;
+
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+
+import javax.portlet.PortletURL;
+
+/**
+ * @author Roberto Díaz
+ */
+public interface AICreatorOpenAIURLHelper {
```

Comment:
Instead of this URL helper, can we follow the same pattern as sharing? See SharingDropdownItemFactory. I think it is a better idea to have an interface that provided various MenuItem, DropdownItem, etc. than something that exposes base URLs that need to be extended; this approach will avoid callers having to know how to contruct valid URLs from the base one.

Reply:
WIP

Reply:
Done, all this changes are in the first commit (I've removed the helper to avoid noise and simplify Brian's review)